juniper_official/Solutions/Traffic-Blackhole/host-process-cpu-memory.rule

```diff
@@ -0,0 +1,73 @@
+/*
+	* REQ: REQ-222
+	* Rule to collect Host Process CPU and Memory Utilization for Blackhole Detection RCA.
+*/
+healthbot {
+	topic hardware.fpc {
+		rule host-process-cpu-memory {
+			keys [ fpc-name process-name ];
+			synopsis "Host Process CPU and Memory Utilization";
+			description "Host Process CPU and Memory Utilization";
+			sensor sensor-host-process-cpu-memory {
+				synopsis "Host Process CPU and Memory Utilization";
+				description "Host Process CPU and Memory Utilization";
+				open-config {
+					sensor-name /junos/system/linecard/cpu/memory/;
+					frequency 60s;
+				}
+			}
+			field fpc-name {
```

Comment:
For keeping common names, can we rename this to "fpc"? Keys section also should be adjusted accordingly

Reply:
Changed to fpc